src/Nri/Ui/Container/V2.elm

```diff
@@ -120,6 +126,39 @@ css css_ =
     Attribute <| \config -> { config | css = config.css ++ css_ }
 
 
+{-| Equivalent to:
+
+    Container.css
+        [ Css.Media.withMedia [ Nri.Ui.MediaQuery.V1.notMobile ] styles ]
```

Comment:
this doesn't help me understand what this does unless I know what `notMobile` is. Could we describe the API in words instead? (Here and in the others)

Reply:
Done!